wolfcrypt/src/asn.c

```diff
@@ -33580,6 +33680,9 @@ int ParseCRL(DecodedCRL* dcrl, const byte* buff, word32 sz, int verify,
     }
     if (ret == 0) {
     #endif
+        /* Parse and store the issuer name. */
+        dcrl->issuer = (byte*)GetNameFromDer((byte*)GetASNItem_Addr(
```

Comment:
Again... This is allocating a buffer too large and copying beyond the provided issuer buffer. This needs to use the actual `length`. It should store in `dcrl->issuerSz` and use it. If size doesn't exist in dcrl then it needs added.

Reply:
Fixed with `dcrl->issuerSz` .